javascript/src/index.ts

```diff
@@ -590,18 +590,28 @@ export interface WebhookUnbrandedRequiredHeaders {
   "webhook-signature": string;
 }
 
+export interface WebhookOptions {
+  raw?: boolean;
+}
+
 export class Webhook {
   private static prefix = "whsec_";
   private readonly key: Uint8Array;
 
-  constructor(secret: string) {
+  constructor(secret: string | Uint8Array, options?: WebhookOptions) {
     if (!secret) {
       throw new Error("Secret can't be empty.");
     }
-    if (secret.startsWith(Webhook.prefix)) {
+    if (!options?.raw && secret instanceof String && secret.startsWith(Webhook.prefix)) {
       secret = secret.substring(Webhook.prefix.length);
     }
-    this.key = base64.decode(secret);
+    if (options?.raw && secret instanceof Uint8Array) {
+      this.key = secret;
+    } else if (options?.raw && secret instanceof String) {
+      this.key = Uint8Array.from(atob(secret.toString()), (c) => c.charCodeAt(0));
```

Comment:
Isn't `atob` the poor man's base64.decode? We shouldn't decode in this situation.